subsys/bluetooth/host/classic/shell/a2dp.c

```diff
@@ -324,6 +324,25 @@ int app_config_req(struct bt_a2dp *a2dp, struct bt_a2dp_ep *ep,
 	return 0;
 }
 
+int app_reconfig_req(struct bt_a2dp_stream *stream,
+	struct bt_a2dp_codec_cfg *codec_cfg, uint8_t *rsp_err_code)
+{
+	*rsp_err_code = 0;
+
+	shell_print(ctx_shell, "receive requesting reconfig and accept");
+	if (*rsp_err_code == 0) {
```

Comment:
Is there some call missing here that should set `*rsp_err_code` to something else than 0? Otherwise the if statement is redundant since you're setting it to 0 right before.

Reply:
Updated. This callback will always accept the `req`, so it set `*rsp_err_code` as 0. remove the redundant codes.